Prototype v1b1.1

ETwilight opened the pull request:
Currently, the code inside onload method can be run properly. If we change the height and width properties at line86 in cgdl-prototype.js to 50 or 100, for example, the picture outside the canvas can been seen. But the picture can still not be shown inside the canvas. I guess there are still some problems with the **drawImage function at line 95**.

ETwilight committed:
Prototype v1b1.1

Currently, the code inside onload can be run properly. If we change the height and width properties at line86 in cgdl-prototype.js to 50 or 100, for example, the picture outside the canvas can been seen. I guess there are still some problems with the drawImage function at line 95.

yechs reviewed (commented):
唔……缩进的话能统一一下风格，不要混用 tab 和空格吗……？

可以沿用C社目前的 code style，js/ts/html/css 全部使用空格，并且缩进两个空格

yechs commented:
功能之类的我可能在一周之内读一下 canvas 的文档看一看

ETwilight committed:
The basic texture module is included in "basic_texture_module.html".

If there are any useless or inefficient code, please help optimize them, thanks.
The document "basic_texture_module.html" supports both Chrome and Edge browser.

ETwilight commented:
> 唔……缩进的话能统一一下风格，不要混用 tab 和空格吗……？
> 
> 可以沿用C社目前的 code style，js/ts/html/css 全部使用空格，并且缩进两个空格

好的好的，才看到

ETwilight commented:
> 功能之类的我可能在一周之内读一下 canvas 的文档看一看

我把bug改掉了，basic_texture_module.html 是可以用的

ETwilight committed:
Updated: the code style, annotations

I used the code style of computerization and added several annotations to the programme to make it more understandable.

ETwilight committed:
Add the "use strict"

See previous commits for detailed description

yechs commented:
我之前咕咕咕了，明天来看一下orz

ETwilight committed:
Update: the class CGDLTextureManager is added to cgdl-prototype.js

ETwilight committed:
Delete basic_texture_module.html
